orally-network/sybil
# Domains removal

Domain authentication logic was removed from the code. + additional cargo warning fixes

How to test:
- deploy canister locally and prepare canister for usage(api_key generation, balance deposit..) 
- open method request link in a browser(so no domains passed as with curl request)
- you should see method output instead of "domain not found" error or code panic
